### DISCUSSION TITLE
How do I get a voice channel by its id?

### DISCUSSION THREAD

**Discussion by Shandeika** [Rapptz/discord.py#6391] [Q&A] [answered]
This can be easily done with a text channel, but how can it be done with a voice channel?

**Comment by bandwiches** [comment, marked as answer]
The same way you do it for a text channel.

[Guild.get_channel()](https://discordpy.readthedocs.io/en/latest/api.html#discord.Guild.get_channel) `returns list [abc.GuildChannel]`
[Guild Channel](https://discordpy.readthedocs.io/en/latest/api.html#discord.abc.GuildChannel) implements one of... `abc.TextChannel`, `abc.VoiceChannel`, `abc.CategoryChannel`.

So maybe in its simplest form, something like this:
```
channel = ctx.guild.get_channel(channel_id)
```
